pages/common/yesod.md

```diff
@@ -0,0 +1,19 @@
+# yesod
+
+> Yesod web framework CLI helper.
```

Comment:
`yesod` here refers to the CLI tool, rather than the framework itself so I'm afraid that would be a bit misleading.